File: src/main/java/com/telegram/codex/conversation/domain/update/ProcessedUpdateRecord.java
package com.telegram.codex.conversation.domain.update;

public record ProcessedUpdateRecord(
    long updateId,
    String chatId,
    long messageId,
    long processedAt,
    String replyText,
    String conversationState,
    String suggestedReplies,
    Long sentAt
) {
}

File: src/main/kotlin/com/telegram/codex/conversation/domain/ProcessedUpdateRecord.kt
package com.telegram.codex.conversation.domain

data class ProcessedUpdateRecord(
    val updateId: Long,
    val chatId: String?,
    val messageId: Long,
    val processedAt: Long,
    val replyText: String?,
    val conversationState: String?,
    val suggestedReplies: String?,
    val sentAt: Long?,
)
